src/components/userProfile/RequestPage.js

```diff
@@ -68,7 +68,7 @@ class RequestPage extends Component {
     if (error) {
```

Comment:
Hmm.. sorry! I have no idea about this. I just add parameter in 'Action.requestSent' method call.